### DISCUSSION TITLE
Welcome to flutter_ui_kit Discussions!

### DISCUSSION THREAD

**Discussion by chandansgowda** [chandansgowda/flutter_ui_kit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chandanaa674-debug** [comment]
Hi bro I see your today YouTube video then check your repositorys I am 1 year aiml student I have no idea how to start anything pls guide me plzs
